go.mod

```diff
@@ -0,0 +1,3 @@
+module go.opentelemetry.io/proto
```

Comment:
I think `go.opentelemetry.io/proto` could work, but `go.opentelemetry.io/otlp` might be more descriptive of what the contents is.

Reply:
So I'm happy to change that, but I would also recommend that we would include `proto` somewhere in the path.  Just to keep the expectation that you won't find any kind of exporter logic or other such affordances in these packages.  They are just the protobuf serde code, and the client and server stubs.

Reply:
From the SIG meeting, the proposed module name is `go.opentelemetry.io/proto/otlp`